tests/views.py

```diff
@@ -39,6 +39,8 @@ def get(self, request, *args, **kwargs):
 
 
 class APITestView(APIView):
+    test_arg = False
```

Comment:
While the fix is OK, this additional line alone does not actually seem to test that the fix works. Maybe you forgot to commit the actual test?